### PR TITLE
#224 simplify maven setup

### PR DESCRIPTION
**Pull request by kaklakariada**

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -4,11 +4,11 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.itsallcode</groupId>
     <artifactId>openfasttrace-parent</artifactId>
-    <version>2.2.0</version><!-- PRODUCT VERSION: do not remove this comment! -->
+    <version>${revision}</version>
     <name>OpenFastTrace Parent</name>
     <packaging>pom</packaging>
     <properties>
-        <oft.version>2.2.0</oft.version>
+        <revision>2.3.0-SNAPSHOT</revision>
```
Comment (redcatbear) [inline parent/pom.xml]: Why is there a SNAPSHOT version here?

File: parent/pom.xml [hunk 2]
```diff
@@ -429,6 +472,25 @@
                                         <ignore></ignore>
                                     </action>
                                 </pluginExecution>
+                                <pluginExecution>
+                                    <pluginExecutionFilter>
+                                        <groupId>
```
Comment (redcatbear) [inline parent/pom.xml]: Please correct (unify) XML formatting.

Comment (redcatbear) [reply parent/pom.xml]: `<groupId>org.codehaus.mojo</groupId>` and so on.

Comment (kaklakariada) [reply parent/pom.xml]: OK. Can you please provide the XML formatter settings so that we can document this?

**Pull request merged**